CMakeLists.txt

```diff
@@ -16,7 +27,7 @@ option(NONET "Disable network" OFF)
 option(RUN_TESTS "Build and run tests" OFF)
 option(HELLFIRE "Build hellfire version" OFF)
 RELEASE_OPTION(CPACK "Configure CPack")
-
+
```

Comment:
Why does GitHub show a diff here I wonder?

Reply:
removed space

Reply:
Different lineends, sorry. Fixed that.